app/src/main/java/com/example/rholbrook/tickettoride/game/GameActivityModel.java

```diff
@@ -174,19 +176,23 @@ public void selectFaceDownCardDeck() {
 
     public void drawTickets() {
         List<Ticket> tickets = client.getTickets();
+        List<Ticket> newTickets = new ArrayList<>();
+        boolean duplicate = false;
         int count = 0;
         int i = 1;
         while (count < 3) {
             for (Ticket ticket : tickets) {
                 if (i == ticket.getTicketId()) {
-                    break;
+                    duplicate = true;
                 }
             }
-            //tickets.add(new Ticket(i, "AwesomeTown", "Blaineville", 1000000));
-            i++;
-            count++;
+            if (!duplicate) {
+                newTickets.add(new Ticket(i, "AwesomeTown", "Blaineville", 1000000));
+                i++;
+                count++;
+            }
```

Comment:
Is this data that you create for the demo?

Reply:
Yeah it is